components/common/Layout/Layout.tsx

```diff
@@ -25,28 +25,33 @@ const Loading = () => (
   </div>
 )
 
-const dynamicProps = {
-  loading: Loading,
```

Comment:
You should keep this assignment, then use the spread syntax `{ ...dynamicProps}` instead of `dynamicProps`. 

That way, you can have common `dynamicProps` without repeating yourself if you need to add some other properties than `loading`. 

It makes the solution easier to maintain.

Reply:
Yes, that's smarter. :+1: